what4/src/What4/Expr/BoolMap.hs

```diff
@@ -179,3 +207,61 @@ reversePolarities (BoolMap m) = BoolMap $! fmap negatePolarity m
 removeVar :: OrdF f => BoolMap f -> f BaseBoolType -> BoolMap f
 removeVar InconsistentMap _ = InconsistentMap
 removeVar (BoolMap m) x = BoolMap (AM.delete (Wrap x) m)
+
+--------------------------------------------------------------------------------
+-- ConjMap
+
+-- | A 'BoolMap' representing a conjunction.
```

Comment:
If you could, please also add some documentation to `BoolMapView` stating that it is equally capable of representing conjunctions and disjunctions, and that if you know you are dealing with a conjunction, you may want to consider using `ConjMap` and friends for clarity.